test/typescript/components.spec.tsx

```diff
@@ -346,9 +346,9 @@ const GridTest = () =>
 
 const GridListTest = () =>
   <GridList cellHeight={160} cols={3}>
-    <GridListTest cols={1}>
+    <GridList cols={1}>
       <img src="img.png" alt="alt text" />
-    </GridListTest>,
+    </GridList>,
```

Comment:
Turning on `noImplicitAny` caught this bug.

Reply:
`GridListTile` instead